test_commit.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+
+def test_commit(): #pragma: no cover
```

Comment:
Not tracked by `coverage` during regular testing, because we don't want to add noise to the git history when troubleshooting.